packages/transaction-controller/src/TransactionController.ts

```diff
@@ -342,6 +383,22 @@ export class TransactionController extends BaseController<
     this.getSelectedAddress = getSelectedAddress;
     this.securityProviderRequest = securityProviderRequest;
 
+    this.afterSign = hooks?.afterSign ? hooks.afterSign : () => true;
+    this.beforeApproveOnInit = hooks?.beforeApproveOnInit
+      ? hooks.beforeApproveOnInit
+      : /* istanbul ignore next */
+        () => true;
+    this.beforeCheckPendingTransaction = hooks?.beforeCheckPendingTransaction
+      ? hooks.beforeCheckPendingTransaction
+      : /* istanbul ignore next */
+        () => true;
+    this.beforePublish = hooks?.beforePublish
+      ? hooks.beforePublish
+      : () => true;
+    this.getAdditionalSignArguments = hooks?.getAdditionalSignArguments
+      ? hooks.getAdditionalSignArguments
+      : () => [undefined];
```

Comment:
It's probably functionally harmless but is the default not `[]` to provide no additional arguments?